scripts/scil_fibertube_tracking.py

```diff
@@ -0,0 +1,287 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Local streamline tractography based on the architecture of
+scil_tracking_local_dev.py, adapted for fibertube tracking.
+
+Void of the concept of grid, voxels and resolution. Instead, the tracking
+algorithm is executed directly on fibertubes (Virtual representation of
+axons). To simulate a lower resolution, a blur_radius parameter forming a
+"tracking sphere" is introduced. At each step, a random direction will be
+picked from the fibertube segments intersecting with the tracking sphere.
+
+Algorithm type is inherently probabilistic with a distribution weighted by the
+volume of intersection between each fibertube segment and the tracking sphere.
+
+The tracking direction is chosen in the aperture cone defined by the
+previous tracking direction and the angular constraint.
+
+Seeding is done within the first segment of each fibertube.
```

Comment:
You can copy-paste from your demo and make sure to refer to it here in this help.